### PR TITLE
[#64] named commands

### LINKED ISSUE #64: Named Commands in auto

### PR DESCRIPTION
**Pull request by Meh243**
Closes #64

Changed the order of initialization so autobuilder is initialized before namedcommands.

### CODE REVIEW

File: src/main/deploy/pathplanner/autos/Spin.auto [hunk 1]
```diff
@@ -14,7 +14,7 @@
         {
           "type": "path",
           "data": {
-            "pathName": "Spin"
+            "pathName": "Sipnslow"
```
Comment (WindowsVistaisCool) [inline src/main/deploy/pathplanner/autos/Spin.auto:17]: sipnslow????

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -90,7 +88,17 @@ protected void initializeSubsystems() {
 	}
 
 	@Override
-	protected void configureButtonBindings() { //TODO decide on comp buttons
+	protected void initializeNamedCommands() {
+		NamedCommands.registerCommand("test", new InstantCommand(() -> System.err.println("HELLO WORLD AHHHHHHHHHHHHH!!!!")));
```
Comment (WindowsVistaisCool) [inline src/main/java/frc/robot/RobotContainer.java:92]: delete plz 🥺

**Review by WindowsVistaisCool — CHANGES_REQUESTED** [review]

**Comment by Meh243** [comment]
i did the problem

**Review by WindowsVistaisCool — APPROVED** [review]
ok

**Pull request merged**